src/scripts/Selector/SelectorTable.js

```diff
@@ -65,22 +65,20 @@ export default class SelectorTable extends Selector {
 		result = Array.from({
 			length: dataCells.length / Object.keys(columnIndices).length,
 		}).map(_ => Object());
+		const offset = dataCells[0].cellIndex; // cellIndex = порядковому номеру элемента среди его одноуровневых элементов
+		const revColInd = Object.entries(columnIndices).map(([k, v]) => [v, k]); // Приводим к entries, а затем инвертируем объект columnIndices
+		const ind2names = Object.fromEntries(revColInd); // Получаем из инвертированного revColInd итерируемый список
```

Comment:
Since we have sugar.js dependency, maybe we can use [this handy function](https://sugarjs.com/docs/#/Object/invert) from it